lbrynet/lbrynet_daemon/UIManager.py

```diff
@@ -61,9 +61,12 @@ def __init__(self, root):
                 self.loaded_requirements = None
 
     def setup(self, branch=None, check_requirements=None, user_specified=None):
-        local_ui_path = settings.local_ui_path or user_specified
-        self.branch = settings.ui_branch or branch
-        self.check_requirements = settings.check_ui_requirements or check_requirements
+        local_ui_path = (user_specified if user_specified is not None
```

Comment:
Fair enough. Fixed. I retained the "is not None" style for user_specified because it's a boolean.